src/Features/Core/Portable/PdbSourceDocument/IPdbSourceDocumentLoaderService.cs

```diff
@@ -13,5 +13,9 @@ internal interface IPdbSourceDocumentLoaderService
         Task<SourceFileInfo?> LoadSourceDocumentAsync(string tempFilePath, SourceDocument sourceDocument, Encoding encoding, IPdbSourceDocumentLogger? logger, CancellationToken cancellationToken);
     }
 
-    internal sealed record SourceFileInfo(string FilePath, TextLoader Loader);
+    /// <param name="FilePath">The path to the source file on disk</param>
+    /// <param name="SourceDescription">The description of where the file came from, for the document tag</param>
```

Comment:
Can you add an example for `SourceDescription`? Since they're coming from disk it's a little confusing what this is actually for